cmd/mysterium_client/command_run/options.go

```diff
@@ -24,6 +25,12 @@ func ParseArguments(args []string) (options CommandOptions, err error) {
 		".",
 		"Runtime directory for temp files (should be writable)",
 	)
+	flags.StringVar(
+		&options.LocalApiAddress,
+		"localApi.port",
```

Comment:
- `tekilaApi.host` - string, defaults to `"localhost"`
- `tekilaApi.port` - int, defaults to `8000`